### PR TITLE
Add Academy Service, Dto, Controller and Technology Service, Dto

### PR DESCRIPTION
**Pull request by BohdanRomaniuk**
Finish working with Academy Flow by creating Groups Service, Dto, Controller and Technology Service and Dto
Add mappers for Academy  and Technology Dto
Try to use Academy with Include and mapping

### CODE REVIEW

File: RAS.Services/Implementation/GroupsService.cs [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+using RAS.DataAccess.Interfaces;
+using RAS.DataAccess.Models;
+using RAS.Services.DTO;
+using RAS.Services.Interfaces;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace RAS.Services.Implementation
+{
+    public class GroupsService: Service, IGroupsService
```
Comment (dimkijeee) [inline RAS.Services/Implementation/GroupsService.cs]: Please, check new implementation of class Service.cs (It`s generic class).

File: RAS.Services/Implementation/GroupsService.cs [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+    {
+        public GroupsService(IUnitOfWork unitOfWork):
+            base(unitOfWork)
+        {
+        }
+
+        public static AcademyDto MapToDto(Academy academy)
```
Comment (dimkijeee) [inline RAS.Services/Implementation/GroupsService.cs]: Next methods should be protected overload

File: RAS.Services/Interfaces/IGroupsService.cs [hunk 3]
```diff
@@ -0,0 +1,12 @@
+﻿using System.Collections.Generic;
+using RAS.DataAccess.Models;
+using RAS.Services.DTO;
+
+namespace RAS.Services.Interfaces
+{
+    public interface IGroupsService
+    {
+        IEnumerable<AcademyDto> GetAll();
+        AcademyDto GetById(int id);
```
Comment (dimkijeee) [inline RAS.Services/Interfaces/IGroupsService.cs]: Fix name of method GetById(int id) to Get(int id)

File: RAS.Services/Implementation/GroupsService.cs (hunk 1 above)
Comment (BohdanRomaniuk) [reply RAS.Services/Implementation/GroupsService.cs]: What branch should I look to see that implementation? Dev hasn't generic Service class.

File: RAS.Services/Implementation/GroupsService.cs (hunk 2 above)
Comment (BohdanRomaniuk) [reply RAS.Services/Implementation/GroupsService.cs]: The same as above

**Pull request merged**